frontend/src/components/MapPosition/MapPositionSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { Typography } from '@equinor/eds-core-react'
+import { Mission } from 'models/Mission'
+import styled from 'styled-components'
+
+const MapPositionView = styled.div`
+    display: grid;
+    grid-column: 1/ -1;
+    gap: 1rem;
+`
+
+interface MissionProps {
+    mission?: Mission
+}
+
+export function MapPositionSection({ mission }: MissionProps) {
+    var text
+    if (mission && mission.robot.pose)
```

Comment:
Should we wrap this in a 
useEffect(()={
updateText
}, []) 
to ensure that it only runs once? alternately 

useEffect(()={
updateText
}, [changingPoseVariable])

Reply:
Good question, let us think more about this.

Reply:
As it is written now it works the same way as the tasks update. I would say that is fine for now, and will create an issue to do this through an API call directly as suggested in the next comment.